packages/terra-header/README.md

```diff
@@ -0,0 +1,42 @@
+# Terra Header
+
+
+[![NPM version](https://badgen.net/npm/v/terra-header)](https://www.npmjs.org/package/terra-header)
+[![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
+
+The terra-header component is a styled layout of a header bar.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-core/tree/master/packages/terra-header/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-header`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+<details>
+<summary>View information on this component's peer dependencies.</summary>
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+
+</details>
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## LICENSE
+
+Copyright 2018 - 2019 Cerner Innovation, Inc.
```

Comment:
This should be 2019

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2545/commits/b4c876c6792a868e1528f1d9d08f59de74dff3af#diff-8a134e17aa611974581a20b68ecc4949R36